### PR TITLE
New Modules for NDB features for 1.8.0-beta.1 release

### PR DESCRIPTION
**Pull request by bhati-pradeep**
Modules:
- ntnx_ndb_databases
- ntnx_ndb_databases_info
- ntnx_ndb_time_machines_info
- ntnx_ndb_clusters_info
- ntnx_ndb_slas_info
- ntnx_ndb_clones_info
- ntnx_ndb_profiles_info
- ntnx_ndb_db_servers_info

### CODE REVIEW

File: plugins/module_utils/ndb/databases.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    def get_uuid(
+        self,
+        value,
+        key="name",
+        data=None,
+        entity_type=None,
+        raise_error=True,
+        no_response=False,
+    ):
+        query = {"value-type": key, "value": value}
```
Comment (abhimutant) [inline plugins/module_utils/ndb/databases.py:42]: query has missing params. For detailed instance of database , query should be like "/databases?detailed=true&load-dbserver-cluster=true"

**Review by Gevorg-Khachatryan-97 — APPROVED** [review]

**Pull request merged**